fast64_internal/oot/__init__.py

```diff
@@ -58,6 +58,13 @@
     oot_operator_unregister,
 )
 
+
+feature_set_enum = (
+    ("decomp", "Decomp", "Decomp"),
+    ("hacker_oot", "HackerOoT", "Hacker OoT"),
```

Comment:
I'm not sure about these names, how about "OoT" instead of "Decomp" instead? technically both repos are decomps so it bothers me to name one "decomp" when they both are, 

now that I think more about it, this is more concerning if the plan is add SoH support since this is also a decomp of OoT, so it should be like "Default", "HackerOoT" and "Ship" (or "SoH" or whatever name you like)

also ``hacker_oot`` should be changed to ``hackeroot`` imo but it's probably fine if this is to help people reading this

Reply:
Done.